plugins/inputs/pf/pf_nocompile.go

```diff
@@ -0,0 +1,3 @@
+// +build !freebsd
```

Comment:
Let's remove all the build flags and always build the plugin.

Reply:
had repeated what had been done in the iptables plugin, but makes sense as pf is on more than FreeBSD.